apps/folder/pkg/apis/folder/v1beta1/types.go

```diff
@@ -44,10 +44,11 @@ type FolderInfo struct {
 type FolderAccessInfo struct {
 	metav1.TypeMeta `json:",inline"`
 
-	CanSave   bool `json:"canSave"`
-	CanEdit   bool `json:"canEdit"`
-	CanAdmin  bool `json:"canAdmin"`
-	CanDelete bool `json:"canDelete"`
+	CanSave       bool            `json:"canSave"`
+	CanEdit       bool            `json:"canEdit"`
+	CanAdmin      bool            `json:"canAdmin"`
+	CanDelete     bool            `json:"canDelete"`
+	AccessControl map[string]bool `json:"accessControl,omitempty"`
```

Comment:
What is the reasoning behind adding this?

Reply:
They're included on the individual details' endpoint, if you pass in `accesscontrol=true` query param, e.g.

`/api/folders/edx2eifh2qx34e?accesscontrol=true`